lib/job-iteration/csv_enumerator.rb

```diff
@@ -30,19 +30,23 @@ def initialize(csv)
     # Constructs a enumerator on CSV rows
     # @return [Enumerator] Enumerator instance
     def rows(cursor:)
+      drop = cursor.nil? ? 0 : cursor + 1
```

Comment:
Could we extract this to a method that expresses the intent of this logic? 

`advance_cursor`?
`move_cursor`?

I'm not very good with names if I'm honest 😄